backend/app/services/codegraph/repo_service.py

```
@@ -0,0 +1,285 @@
+                            "cypher": data.get("cypher_query", "")
+                        }
+                    else:
+                        error_text = (await response.text())[:200]
+                        logger.error(f"Query failed [{response.status}]: {error_text}")
+                        return {
+                            "status": "error",
+                            "message": "Query failed. Please try again or rephrase your question."
+                        }
+
+        except aiohttp.ClientError as e:
+            logger.error(f"Network error querying {repo_full_name}: {e}")
```

Ruff (0.13.3)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)